test/test_parallel_executor.cpp

```diff
@@ -340,248 +282,6 @@ TEST(ParallelExecutor, ExecuteOnSingleThread8Bit) {
   // Compare(tf_output_tensor, expected_val, 0);
 }
 
-TEST(ParallelExecutor, ExecuteOnMultipleThreads8Bit) {
```

Comment:
These tests needn't be removed right. They will look different under the new circumstances, but they should be essentially testing the things they were testing earlier since these tests are kind of end-to-end tests

Reply:
You could do this (or the next) test with 1, 2 and 3 threads. the 3 threads one is expected to fail (of course we may have to barrier it so that we ensure that all 3 threads are running at the same time so that the PTS is empty for atleast one of them.